Source/WebKit/Scripts/webkit/tests/TestWithCVPixelBufferMessages.h

```diff
@@ -56,9 +56,9 @@ class SendCVPixelBuffer {
     {
     }
 
-    const auto& arguments() const
+    auto&& arguments()
```

Comment:
The point of these tests is to test the code you modify.
So you could add a thing that shows how your new changes behave.